packages/components/select/__tests__/select.spec.ts

```diff
@@ -315,6 +315,35 @@ describe('Select', () => {
       expect(input.element.style.opacity).toBe('0')
     })
 
+    test('issue #750', async () => {
```

Comment:
```suggestion
    test(searchFilter with labelKey work, async () => {
```
另外把这个测试用例放到  `searchFilter work` 的测试用例后面吧。